src/components/data-display/tables/TableView.tsx

```diff
@@ -161,7 +163,7 @@ const TableView = <T, S>({
               </tr>
             ))}
           </thead>
-          <tbody className="text-sm">
+          <tbody className={`${bodyClass} text-sm`}>
```

Comment:
same here

Reply:
yup this was a deliberate change. Prepending ensures overridability of the default classes but will still add them if not conflicting.